### PR TITLE
Fix name of Java options environement variable

### LINKED ISSUE #528: Update helm chart proxy guide
[Guide for running Jenkins behind proxy](https://github.com/jenkinsci/helm-charts/blob/main/charts/jenkins/README.md#running-behind-a-forward-proxy) appears to be incorrect.

Should use `JAVA_OPTS` instead of `JAVA_OPTIONS`.

Tested with latest chart version, reproducible in every version.

### PR DESCRIPTION
**Pull request by gbonnefille**
The variable expected by jvm is prefixed by underscore.

### Which issue this PR fixes

*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*

- fixes #528

### Checklist

- [ ] [DCO](https://github.com/jenkinsci/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [ ] Chart Version bumped
- [ ] CHANGELOG.md was updated

### CODE REVIEW

**Comment by gbonnefille** [comment]
It is only a document change... not sure I have to bump the version for that.

**Review by timja — CHANGES_REQUESTED** [review]

File: charts/jenkins/README.md [hunk 1]
```diff
@@ -853,7 +853,7 @@ controller:
       value: "http://192.168.64.1:3128"
     - name: no_proxy
       value: ""
-    - name: JAVA_OPTIONS
+    - name: _JAVA_OPTIONS
```
[the remark this answers is not in the document]
Comment (timja) [reply charts/jenkins/README.md:856]: do not use that value, see https://stackoverflow.com/a/40736904/4951015

[the remark this answers is not in the document]
Comment (gbonnefille) [reply charts/jenkins/README.md]: OK

**Comment by charlesmelby** [comment]
It would be great if this PR could be completed.
I just wasted a couple hours trying to figure out this issue myself.

**Comment by timja** [comment]
Few free to submit a new PR

**Comment by torstenwalter** [comment]
Can you fix the DCO check and maybe update the title of the PR?

**Comment by gbonnefille** [comment]
> Can you fix the DCO check and maybe update the title of the PR?

I added signed-of on all my commits.

**Comment by timja** [comment]
and resolve the conflicts please?

**Comment by gbonnefille** [comment]
> and resolve the conflicts please?

Done, but please, merge it quicly: it is a HUGE effort for a so trivial fix.

**Review by timja — APPROVED** [review]

**Pull request merged**